pkgs/ok_http/README.md

```diff
@@ -12,22 +12,43 @@ An Android Flutter plugin that provides access to the
 
 This size of the [example application][] APK file using different packages:
 
-| Package                                                                                   | APK Size (MiB) |
-| ----------------------------------------------------------------------------------------- | -------------- |
-| **`ok_http`**                                                                             | **20.3**       |
-| [`cronet_http`](https://pub.dev/packages/cronet_http) [^1]                                | 20.6           |
-| [`cronet_http` (embedded)](https://pub.dev/packages/cronet_http#use-embedded-cronet) [^2] | 34.4           |
-| `dart:io` [^3]                                                                            | 20.4           |
+| Package | APK Size (MiB) |
+|-|-|
+| **`ok_http`** | **20.3**  |
+| [`cronet_http`](https://pub.dev/packages/cronet_http) [^1] | 20.6 |
+| [`cronet_http` (embedded)](https://pub.dev/packages/cronet_http#use-embedded-cronet) [^2] | 34.4 |
+| `dart:io` [^3] | 20.4 |
 
 [^1]: Requires [Google Play Services][], which are not available on all devices.
 [^2]: Embeds the Cronet HTTP library.
 [^3]: Accessed through [`IOClient`](https://pub.dev/documentation/http/latest/io_client/IOClient-class.html).
 
 ### 🔌 Supports WebSockets out of the box
 
-`package:ok_http` wraps the OkHttp [WebSocket][] API, allowing you to use it in your Flutter app with ease.
+`package:ok_http` wraps the OkHttp [WebSocket][] API which supports:
 
-See the [OkHttp Example App][] for usage notes.
+- Configured System Proxy on Android
+- HTTP/2
+
+```dart
+import 'package:ok_http/ok_http.dart';
```

Comment:
That's a nice point! Here the example is as minimal/boring as it can get, mostly to demonstrate how conformant its usage is, in comparison to other WebSockets. See PR https://github.com/dart-lang/http/pull/1266 that added the same example to `CupertinoWebSocket`, which was originally taken from the `README` of `package: web_socket`. This package contains a better example of WebSockets (Cryptocurrency API)

If you still consider that a more comprehensive example is suitable for this as well, I can start working on it!